### PR TITLE
Connection: update design in full connection view

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request:

* Updates the full screen connection view with some design tweaks:
  * New, more consistent looking illustrations.
  * Larger type for better legibility.
  * New subsection headings to highlight each major benefit.

cc @robertbpugh so you can weigh in whether we need better copy as suggested before.

#### Jetpack product discussion

See p6TEKc-3MH-p2 for more details. Kudos @Luchadores for the refreshed design!

#### Testing instructions:

* Spin up this PR on a site that is disconnected. Either start with a brand new site or disconnect your existing site.
* Click on `Jetpack` on the wp-admin sidebar.
* Ensure you see the new design on the full connection view and that everything looks as expected.

#### Screenshots

Before | After
--|--
image |image
image | image

#### Proposed changelog entry for your changes:
* None

### CODE REVIEW

**Comment by folletto** [comment]
Looks sharper to me. There's a better balance and it's easier to follow.

One note: can't we use font variables yet instead of using `rem` values?

**Comment by keoshi** [comment]
@folletto Glad you noticed that! :D Yes, we totally should but we're still waiting on that PR to be merged. I'm holding off on this one until that's done and will note that in the description too.

**Review by Luchadores — COMMENTED** [review]
Looking great!

**Review by crunnells — DISMISSED** [review]
Nice improvement here! These changes make this dialog more engaging and easier to read..

File: class.jetpack-connection-banner.php [hunk 1]
```diff
@@ -352,45 +352,39 @@ class="dops-button is-primary jp-connect-button">
...
+							<ul>
+								<li><?php esc_html_e( 'Keep people on your site longer with lightning-fast page load times through our free global CDN.', 'jetpack' ); ?></li>
+								<li><?php esc_html_e( 'Speed up your mobile site and reduce bandwidth usage automatically.', 'jetpack' ); ?></li>
+								<li><?php esc_html_e( 'Improve visitor engagement and sales with a customized search experience.', 'jetpack' ); ?></li>
+							</ul>
 						</div>
 					</div>
 				</div>
 
+				<h2><?php esc_html_e( 'More than 5 million WordPress sites trust Jetpack for their website security and performance', 'jetpack' ); ?></h2>
```
Comment (jeherve) [inline class.jetpack-connection-banner.php:386]: Should this end with a period?

```suggestion
				<h2><?php esc_html_e( 'More than 5 million WordPress sites trust Jetpack for their website security and performance.', 'jetpack' ); ?></h2>
```

**Review by jeherve — DISMISSED** [review]
This looks good to me.

Part of me wonders if the changes could have a negative impact, since the page is a bit longer and the button is a bit downer. Do you think that's minor enough to not really be an issue? I could imagine that on phones with a small screen, the button could end up being off-screen now?

Screenshot_20200731-114352
Screenshot_20200731-114309

I also have a tiny piece of feedback, but I don't think this should be a blocker.

**Comment by keoshi** [comment]
Good feedback, @jeherve ! @Luchadores and I discussed this and it feels like a no brainer to tweak as you suggested. Will push a few changes in a minute.

**Comment by keoshi** [comment]
Thanks for that again, @jeherve — as you can see in the screenshots above, we even managed to bump the button higher up while increasing legibility!

**Review by jeherve — APPROVED** [review]
💥 Nice work!

**Pull request merged**